game-core/src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -678,8 +679,8 @@ private static void usage() {
   private static void handleHeadlessGameServerArgs() {
     boolean printUsage = false;
 
-    final String mapFolder = ClientSetting.MAP_FOLDER_OVERRIDE.value();
-    if (mapFolder.isEmpty() || !(new File(mapFolder).exists()) || !(new File(mapFolder).isDirectory())) {
+    final File mapFolder = ClientFileSystemHelper.getUserMapsFolder();
+    if (!mapFolder.isDirectory()) {
```

Comment:
Note that `File#isDirectory()` implies `File#exists()`.  Per the Javadocs:

> `true` if and only if the file denoted by this abstract pathname exists _and_ is a directory; `false` otherwise